machida/machida.pony

```diff
@@ -326,7 +326,7 @@ class PyComputation is Computation[PyData val, PyData val]
     let r: Pointer[U8] val =
       Machida.computation_compute(_computation, input.obj(), _is_multi)
 
-    if not r.is_null() then
```

Comment:
We still need to do the null check here, because the return value will be null if the computation raised an exception. If it is null we should fail.